src/pages/folder/Index.tsx

```diff
@@ -1,53 +1,179 @@
-import { useEffect, useState } from "react";
 import PostCard from "../../components/folder/PostCard";
-import SearchInput from "../../components/form/FormInput";
-import { ContainBody, ContainHead, TitleMs } from "../../styles/commonStyle";
+import { ContainBody, ContainHead, SubTitle } from "../../styles/commonStyle";
 import {
-  FolderContainBodyInner,
-  FolderContainHeadInner,
+  BookMarkBtnList,
+  BookmarkBox,
+  BodyInner,
+  ShareBox,
+  EmptyBox,
+  BoxLinkSearch,
+  ShareListBtn,
   PostCardWrap,
+  LinkAddHeadInner,
 } from "./folderStyle";
-import { IFolderListApi, folderListApi } from "../../constant/api";
-const logo = "/assets/logo/logo_codeit.svg";
+
+import Button from "../../components/common/atoms/Button";
+import Input from "../../components/common/atoms/Input";
+import useFetch from "../../hook/useFetch";
+import { FOLDERCONTANTLISTAPI, FOLDERMENULISTAPI } from "../../constant/api";
+import { IFolderContentApi, IFolderMenuButtonApi } from "./interface";
+import { useEffect, useState } from "react";
+
+const add = "/assets/icon/icon_primary_add.svg";
```

Comment:
아이콘은 컴포넌트처럼 대문자로 써주시는 게 가독성이 좋아요~

```suggestion
const AddIcon = "/assets/icon/icon_primary_add.svg";
```